transitions/core.py

```diff
@@ -334,6 +337,7 @@ def __init__(self, model=None, states=None, initial=None, transitions=None,
                 executed in a state callback function will be queued and executed later.
                 Due to the nature of the queued processing, all transitions will
                 _always_ return True since conditional checks cannot be conducted at queueing time.
+            add_self (boolean): If no model(s) provided, intialize state machine against self.
```

Comment:
@tyarkoni Makes sense. For the future I still think we should assume most uses will have a single model and opt not to break the API, but we can figure that out at that point and don't need to go too much into it here.

Thanks @paulbovbel! Nice clean implementation.